components/CMakeLists.txt

```diff
@@ -93,7 +93,7 @@ IF(NOT WIN32 AND NOT APPLE)
     add_definitions(-DGLOBAL_CONFIG_PATH="${GLOBAL_CONFIG_PATH}")
 ENDIF()
 add_component_dir (files
-    linuxpath androidpath windowspath macospath fixedpath multidircollection collections configurationmanager
+    linuxpath androidpath windowspath macospath fixedpath multidircollection collections configurationmanager escape
```

Comment:
On my machine, this seemed to be enough to make everything find the new file once CMake was re-run, but the CI systems can't.